### ISSUE TITLE
"AttributeError: 'CHRBND' object has no attribute 'FLVER_Class'" trying to export a character in Demon's Souls 

### ISSUE THREAD

**Issue by thegreatgramcracker** [Grimrukh/soulstruct-blender#77]
Hello, I'm getting a python error when I try to export an imported character model. This error happens with or without any changes to the imported model.
```
Python: Traceback (most recent call last):
  File "C:\Users\thegr\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\io_soulstruct\flver\models\export_operators.py", line 409, in execute
    model_stem, chrbnd, flver, textures = self.get_binder_and_flver(
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thegr\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\io_soulstruct\flver\models\export_operators.py", line 345, in get_binder_and_flver
    binder = binder_class.from_path(binder_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thegr\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\io_soulstruct_lib\soulstruct\containers\core.py", line 379, in from_path
    binder = cls.from_bytes(reader, bdt_reader)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thegr\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\io_soulstruct_lib\soulstruct\containers\core.py", line 307, in from_bytes
    instance = cls.from_reader(reader)
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thegr\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\io_soulstruct_lib\soulstruct\containers\core.py", line 419, in from_reader
    return cls(entries=entries, is_split_bxf=bdt_reader is not None, **header_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 16, in __init__
  File "C:\Users\thegr\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\io_soulstruct_lib\soulstruct\containers\binder_types.py", line 45, in __post_init__
    self.flvers[entry.minimal_stem] = entry.to_binary_file(self.FLVER_CLASS)
                                                           ^^^^^^^^^^^^^^^^
AttributeError: 'CHRBND' object has no attribute 'FLVER_CLASS'
```
There's no other warning or error messages before or after this.

Also, in order to get this far, I needed to copy the chrbnd.dcx from the folder of the character I wanted to edit in to the root of the `chr` directory. I think it might be expecting chrbnds to be in this folder when actually the structure is more like DS:PTDE where each chrbnd is in its own folder.

Another small issue: I can't seem to find a way to export into a chrbnd.dcx. If you choose to export into a binder and select a dcx binder, it goes to the non DCX version instead.

Thank you!

**Comment by Grimrukh** [comment]
This should be fixed in 2.1.6, just a typo on my part.

**Comment by Grimrukh** [comment]
Also fixed the DeS CHRBND detected location!
